.github/scripts/check-gatus-coverage.py

```
@@ -339,6 +452,65 @@ def self_test() -> int:
+        "  evil-svc: {networks: {digit: {aliases: [postgres-db]}}}\n",
+    )
+
+    # 8d. a legitimate alias (postgres -> pgbouncer) must still be accepted, and
+    #     the same alias repeated across overlay files is not a collision.
+    with tempfile.TemporaryDirectory() as td:
+        a = pathlib.Path(td) / "docker-compose.a.yml"
+        b = pathlib.Path(td) / "docker-compose.b.yml"
+        a.write_text("services:\n  pgbouncer: {ports: ['5432'], networks: {digit: {aliases: [postgres]}}}\n")
+        b.write_text("services:\n  pgbouncer: {networks: {digit: {aliases: [postgres]}}}\n")
+        try:
+            t, l = compose_targets([a, b])
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)